addon/components/item-picker/component.js

```diff
@@ -82,6 +82,15 @@ export default Component.extend({
     return componentName;
   }),
 
+  /**
+   * Determine what preview component to use. This allows us to create
+   * per-type UX for the preview
+   */
+  row: computed('rowComponent', function () {
+    let row = this.get('rowComponent');
+    return !row ? 'item-picker/item-row' : row;
```

Comment:
I think `return this.getWithDefault('rowComponent', 'item-picker/item-row')` is a bit easier to grok. Either that or

```
const row = this.get('rowComponent');
return row || 'item-picker/item-row';
```